rusk/src/lib/lib.rs

```diff
@@ -143,10 +159,13 @@ impl Rusk {
                 Ok(call_result) => call_result,
                 Err(err) => match err {
                     piecrust::Error::OutOfPoints => {
+                        let fee_limit =
+                            compute_gas_limit(tx.fee.gas_limit, u64::MAX);
+
                         // If the transaction would have been out of points
                         // with its own gas limit, it is invalid and should
                         // be discarded.
-                        if gas_limit == tx.fee.gas_limit {
```

Comment:
Not related to this PR, but this may be the culprit of why we don't see any more OOG error. I believe that OOG should not be removed if they can spend the fee.

Or I am wrong and a TX that goes OOG is still returning `OK(())`?

Reply:
`OOG` should only happen if the transfer contract goes out of gas. Otherwise it's a `ContractError::OUTOFGAS`. I believe this is what you're seeing.